### DISCUSSION TITLE
Adding Documentation

### DISCUSSION THREAD

**Discussion by JerrettDavis** [awaescher/OllamaSharp#113] [Ideas]
I'd like to start getting some formalized documentation built into the repository. Does anyone have any strong feelings one way or the other regarding tools or documentation in general? 

I was considering [docfx](https://dotnet.github.io/docfx/) as a potential option.

**Comment by awaescher** [comment]
How do you imagine the artifacts could be/look like?

**Reply by JerrettDavis** [reply]
Ideally, the documentation would be generated/live in the /docs/ folder of the repository. Tools like docfx have the ability to generate some of this documentation on build by way of nuget packages. Further, we can have a github action to push the resulting site to Github pages.

**Comment by awaescher** [comment]
Coming back to this after your docs site was merged in 4.0.11, more details [here](https://github.com/awaescher/OllamaSharp/releases/tag/4.0.11).
Once again, a huge thanks for the efforts you put into this!
